public/static/locales/en/manageProjects.json

```diff
@@ -91,5 +91,14 @@
   "backToSpending": "Back to project spending",
   "pendingReview": "Pending Review",
   "submitForReview": "Submit for Review",
-  "addProject": "Add Project"
+  "addProject": "Add Project",
+  "abandonment": "Abandonment",
+  "abandonmentInfo": "Abandonment Information",
```

Comment:
This is probably wrong at should be replace with this like already live at https://beta.plant-for-the-planet.org/yucatan-reforestation
<img width="163" alt="Bildschirmfoto 2020-11-26 um 10 17 56" src="https://user-images.githubusercontent.com/1532418/100331703-13ca6b00-3003-11eb-88e2-9bae418b2728.png">